daemon/oci_linux_test.go

```diff
@@ -174,6 +175,21 @@ func TestSysctlOverrideHost(t *testing.T) {
 	assert.Equal(t, s.Linux.Sysctl["net.ipv4.ip_unprivileged_port_start"], c.HostConfig.Sysctls["net.ipv4.ip_unprivileged_port_start"])
 }
 
+//Test both cases of the default capabilities.
+func TestDefaultCapabilitiesSe(t *testing.T) {
+	c := &container.Container{
+		Config:     &containertypes.Config{},
+		HostConfig: &containertypes.HostConfig{},
+	}
+	d := setupFakeDaemon(t, c)
```

Comment:
You should be able to add a 2nd case and set `d.configStore.DefaultCapabilities` directly.

Reply:
Gotcha thanks.

Reply:
After some goofs I got it working. For the other issue I was seeing, if I let the tests run to completion then re-run the tests I get the same failure.